docs_src/security/Ch-RemoteDeviceServices.md

```diff
@@ -160,7 +160,7 @@ level=INFO ts=2022-05-05T14:29:25.252304305Z app=device-virtual source=devices.g
 level=INFO ts=2022-05-05T14:29:25.252698155Z app=device-virtual source=autodiscovery.go:33 msg="AutoDiscovery stopped: disabled by configuration"
 level=INFO ts=2022-05-05T14:29:25.252726349Z app=device-virtual source=autodiscovery.go:42 msg="AutoDiscovery stopped: ProtocolDiscovery not implemented"
 level=INFO ts=2022-05-05T14:29:25.252736451Z app=device-virtual source=message.go:50 msg="Service dependencies resolved..."
-level=INFO ts=2022-05-05T14:29:25.252804946Z app=device-virtual source=message.go:51 msg="Starting device-virtual 2.2.0-dev.15 "
+level=INFO ts=2022-05-05T14:29:25.252804946Z app=device-virtual source=message.go:51 msg="Starting device-virtual {{latest_released_version}}-dev.15 "
```

Comment:
HUmm, I don't think we want a link to a dev tag. Will be broken once we clean up the tags.
```suggestion
level=INFO ts=2022-05-05T14:29:25.252804946Z app=device-virtual source=message.go:51 msg="Starting device-virtual {{latest_released_version}} "
```